runtime/doc/eval.txt

```diff
@@ -4849,6 +4851,11 @@ getcwd([{winnr} [, {tabnr}]])
 		the window in the specified tab page.
 		Return an empty string if the arguments are invalid.
 
+getenv([{name}])					*getenv()*
+		Return environment variable of {name}. See also |expr-env|.
+		If {name} is not given, return all of environment variables
+		as dictionary.
```

Comment:
Are you going to implement this later?
It seems that this is not implemented yet.

Reply:
Sorry, this was outdated.